packs/astral/actions/get_sunrise.yaml

```diff
@@ -0,0 +1,6 @@
+---
+name: "get_sunrise"
+runner_type: "run-python"
+description: "Returns the time of the sunrise in UTC"
```

Comment:
Added Timezone support.  I looked into using location by city name, but there was some issues in the Astral code for city support. For example, Portland, Maine or Portland, Oregon. It had both as just "Portland".  Plus, I figured it's so easy these days to get your GEO, who cares about the city!